### DISCUSSION TITLE
add more buttons in github jekyll theme

### DISCUSSION THREAD

**Discussion by rudrab** [community/community#21944] [Other Feature Feedback, Questions, & Ideas]
<p>Hi friends,</p>
<p>I have generated a <a href="http://rudrab.github.io/Shadow/" rel="nofollow noopener">github page</a> from my README.md. The _config.yml is:</p>
<pre><code>theme: jekyll-theme-cayman
 	plugins:
 	- jekyll-seo-tag
 	author: Barud
</code></pre>
<p> The problem is at the top, it only links to “View on GItHub”. Unlike in the template, where it has three buttons. I need one more button and link it to my another repo. <br>
How I can do that?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi rudrab,</p>
<p>The Cayman theme has an extra variable that you can include in your _config.yml file to enable these buttons, which will then link to a direct download of a zip of your full repository.</p>
<p>You can switch to the three button mode by including the following line in your _config.yml file:</p>
<pre><code>show_downloads: true
</code></pre>
<p>If you’d like to change where these buttons link to, or modify your theme even further, you can have a read over our documentation on <a href="https://help.github.com/articles/customizing-css-and-html-in-your-jekyll-theme/" rel="nofollow noopener">customizing the CSS and HTML</a> of your theme.</p>
<p>Thanks,</p>
<p>Thomas</p>
